config.yaml

```diff
@@ -2,34 +2,19 @@
 # See LICENSE file for licensing details.
 
 options:
-  # pagerduty config
-  pagerduty::send_resolved:
-    type: boolean
-    description: Whether or not to notify about resolved alerts (default = true).
-    default: true
-  pagerduty::service_key:
+  config_file:
     type: string
-    description: The PagerDuty integration key (when using PagerDuty integration type `Prometheus`).
+    description: >
+      Alertmanager configuration file (yaml), with the exclusion of the templates section.
+      Refer to https://www.prometheus.io/docs/alerting/latest/configuration/ for full details.
     default: ""
-  # pushover config
-  pushover::send_resolved:
-    type: boolean
-    description: Whether or not to notify about resolved alerts (default = true).
-    default: true
-  pushover::user_key:
+  templates_file:
```

Comment:
I wonder if we should call it `templates` and `config` instead. Ofc if we do it like currently in the PR, when Juju finally has structured configs, we still have the cool config names available. I am conflicted.